components/proxy/src/test/java/com/hotels/styx/proxy/backends/file/FileBackedBackendServicesRegistryFactoryTest.java

```diff
@@ -72,9 +105,34 @@ public void requiresOriginsFileToBeNonEmpty() {
         new FileBackedBackendServicesRegistry.Factory().create(environment, configuration);
     }
 
-    private Configuration mockConfiguration(Optional<String> s) {
+    @Test
+    public void fileMonitorIsTurnedOffByDefault() {
+        Environment environment = new com.hotels.styx.Environment.Builder().build();
```

Comment:
If `environment` is not test-specific, it can go in the set-up, to minimise clutter in the tests themselves.